pages/Teams/Player_Profiles/player-profile-2.cfm

```diff
@@ -21,9 +21,12 @@
 </cfquery>
 
 <cfquery name="getPlayerAwards" datasource="roundleague">
-	SELECT AwardName
-	FROM Awards
-	WHERE PlayerID = <cfqueryparam cfsqltype="INTEGER" value="#url.playerID#">
+    SELECT AwardName,awards.seasonID, seasons.SeasonName
+    FROM Awards
+    JOIN 
+    Seasons
+    ON awards.SeasonID = Seasons.SeasonID
+    WHERE PlayerID = <cfqueryparam cfsqltype="INTEGER" value="#url.playerID#">
```

Comment:
```suggestion
    SELECT AwardName, awards.seasonID, seasons.SeasonName
    FROM Awards
    JOIN  Seasons ON awards.SeasonID = Seasons.SeasonID
    WHERE PlayerID = <cfqueryparam cfsqltype="INTEGER" value="#url.playerID#">
```
Formatting the query so it matches other queries, but other then that great job.